src/Pos/Web/Server.hs

```diff
@@ -77,12 +77,13 @@ applicationGT = do
     return $ serve gtNodeApi server
 
 -- [CSL-217]: do not hardcode logStdoutDev.
-serveImpl :: MonadIO m => m Application -> String -> Word16 -> m ()
-serveImpl application host port =
-    liftIO . runSettings mySettings . logStdoutDev =<< application
+serveImpl :: MonadIO m => m Application -> String -> Word16 -> FilePath -> FilePath -> FilePath -> m ()
```

Comment:
Agreed. This was implemented long ago in a big hurry - with no architecture in mind